docs/site/Model.md

```diff
@@ -190,9 +188,188 @@ class Product extends Entity {
 }
 ```
 
-Additionally, the model decorator is able to build the properties object through
-the information passed in or inferred by the property decorators, so the
-properties key value pair can also be omitted.
+However, the model decorator in lb4 is not exactly the same as what it is in
+lb3. For example, in
+[lb3](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html) we can
+pass in a model definition syntax in model decorator, such as properties,
+options, relation etc. But not all these entries are available in lb4 model
+decorator:
+
+In lb4, we only support `properties` and `setting` in the ModelDefinitionSyntax
+for now. Those `top-level properties` in lb3 now are passed in `settings`.
+[Options](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html#options)
+in lb3 doesn't have the mapping feature in lb4 yet. (see
+[issue #2142](https://github.com/strongloop/loopback-next/issues/2142) for
+further discussion.)
+
+As for entries in `settings`, lb4 supports these built-in entries for now:
+
+#### Supported Entries of Settings
+
+<!-- These entries might need to update once we've made some changes:
+  - description [in lb3 we support array or string, but here we documented it as string only]
+  these two don't seem work in lb4. Moved them to unsupported table.
+  - excludeBaseProperties [this feature hasn't been tested yet. ]
+  - base [moved it to unsupported table. Not sure if it actually works in lb4.]-->
+
+  <table>
+  <thead>
+  <tr>
+    <th width="160">Property</th>
+    <th width="100">Type</th>
+    <th>Default</th>
+    <th>Description</th>
+  </tr>
+  </thead>
+
+  <tbody>
+
+  <tr>
+  <td><code>description</code></td>
+  <td>String</td>
+  <td>None</td>
+  <td>
+    Optional description of the model. We only support string type for now. (see <a href="https://github.com/strongloop/loopback-next/issues/3428">issue #3428</a> for more discussion.)
```

Comment:
I thought that repository-json-schema package is not copying the description field to the emitted schema, but apparently it does so ✌️ 

https://github.com/strongloop/loopback-next/blob/27a7e15d87ab14109dbd03e0556fdda31cdb0a0b/packages/repository-json-schema/src/build-schema.ts#L329-L331